modules/javafx.controls/src/main/java/javafx/scene/control/skin/VirtualFlow.java

```diff
@@ -1867,12 +1886,27 @@ private void resizeCellSize(T cell) {
         }
     }
 
-    private List<T> getCells() {
+    /**
+     * Returns the list of cells displayed in the current view.
+     * <p>
+     * The cells are ordered such that the first cell in this list is the first in the view, and the last cell is the
+     * last in the view. When pixel scrolling, the list is simply shifted and items drop off the beginning or the end,
```

Comment:
Could just be my lack of familiarity with this API. I was thinking that if I have a `TableView`, there are cells arranged in a 2D grid (rows and columns). If these are the cells that this methods returns in a (1D) list, how is the reduction happening? If the cells of `getCells` are rows (of cells) then it's fine, though I hope that this concept of virtual flow is known, otherwise calling "rows" "cells" is very confusing.